### DISCUSSION TITLE
Repeated products in gen.py

### DISCUSSION THREAD

**Discussion by anassajaanan** [geeksblabla/blanat#73] [General]
those products are repeated in gen.py file:

"Cranberry"
"Cantaloupe"
"Honeydew"
 "Coconut"
"Pomegranate"
"Artichoke"

![image](https://github.com/geeksblabla/blanat/assets/114141528/38930e94-f075-41cc-90ad-ecab9426f95b)
<img width="569" alt="image" src="https://github.com/geeksblabla/blanat/assets/114141528/12d4aa61-f06b-4da6-a7fa-7f9e3405ca16">

can we have the unique 100 products list that will be used ?

**Comment by chermehdi** [comment]
Not a major a issue :)
